app/javascript/Pages/Home.jsx

```diff
@@ -1,78 +1,132 @@
 import React, { Component } from "react";
 import Navbar from "./components/Navbar/Navbar";
 import Footer from "./components/Footer/Footer";
-import { Head, Link } from "@inertiajs/inertia-react";
-import "../stylesheets/Home.css";
+import { Head } from "@inertiajs/inertia-react";
 import { useState, useEffect } from "react";
 import SocialCard from "./components/SocialCard";
+import "../stylesheets/home.scss";
+import ButtonDifficultyFilter from "./components/ButtonDifficultyFilter";
+import CardCategory from "./components/CardCategory";
+import CardPopular from "./components/CardPopular";
 
 function Homepage(props) {
   const token = document.querySelector("[name=csrf-token]").content;
   const headers = { "X-CSRF-Token": token };
 
-  const [items, setUsers] = useState([]);
+  const [items, setItems] = useState([]);
+  const [cats, setCat] = useState([]);
+  const [pclasses, setPClasses] = useState([]);
+
+  const fetchClassesDifficulty = async (difficulty) => {
+    let newClassData;
+    try {
+      const response = await fetch(`/api/classes?difficulty=${difficulty}`);
+      newClassData = await response.json();
+    } catch (error) {
+      console.log(error);
+      newClassData = [];
+    }
+
+    setItems(newClassData);
+  };
+
+  const fetchCategories = async () => {
+    let categoryData;
+    try {
+      const response = await fetch("/api/skills");
+      categoryData = await response.json();
+    } catch (error) {
+      console.log(error);
+      categoryData = [];
+    }
+
+    setCat(categoryData);
+  };
+
+  const fetchPopularClasses = async () => {
+    let popularClassData;
+    try {
+      const response = await fetch("/api/classes");
+      popularClassData = await response.json();
+    } catch (error) {
+      console.log(error);
+      popularClassData = [];
+    }
+
+    setPClasses(popularClassData);
+  };
 
   useEffect(() => {
-    (async () => {
-      let newClassData;
-      try {
-        const response = await fetch("/api/classes");
-        newClassData = await response.json();
-      } catch (error) {
-        console.log(error);
-        newClassData = [];
-      }
-
-      setUsers(newClassData);
-    })();
+    fetchClassesDifficulty("beginner");
+    fetchCategories();
+    fetchPopularClasses();
   }, []);
   return (
     <main>
+      <Head title="Welcome" />
       <Navbar current_user={props.current_user} />
       <div className="hero-img"></div>
 
-      <Head title="Welcome" />
+      <div className="new-classes">
+        <h1 className="home-title">New classes added every week</h1>
```

Comment:
**question (non-blocking)**: is this website in english or portuguese? Something that could be useful for you to experiement with is setting up "locales" (search for "i18n" as well). That way, instead of having the copy (labels and text) in many files you have only 1 file with all the text and can easily add translations by duplicating the file

Reply:
I've never done it but I'll talk to Galvão tomorrow to see if it's interesting for MVP.